receiver/vcenterreceiver/metrics.go

```diff
@@ -71,6 +72,31 @@ func (v *vcenterMetricScraper) recordVMUsages(
 		diskUtilization := float64(diskUsed) / float64(diskFree+diskUsed) * 100
 		v.mb.RecordVcenterVMDiskUtilizationDataPoint(now, diskUtilization)
 	}
+
+	s := vm.Summary
+	z := s.QuickStats
+
+	cpuUsage := z.OverallCpuUsage
+	var cpuUtilization float64
+
+	ncpu := vm.Config.Hardware.NumCPU
+	if cpuUsage != 0 {
```

Comment:
When `cpuUsage == 0`, I think we shouldn't record the metric, unless we think it really is 0.

Reply:
You mean move the `RecordVcenterVMCPUUsageDataPoint`  and ` RecordVcenterVMCPUUtilizationDataPoint` inside the `if != 0`?

Reply:
I think that's appropriate, if you agree these "0"'s are meaningful

Reply:
Should there be cpu usage per vm in the simulator? Running it locally only gives me 0 for usage.